src/redux/dragons/dragons.js

```diff
@@ -1,6 +1,30 @@
+import axios from 'axios';
+
+const DRAGONS_FETCHED = 'DRAGONS_FETCHED';
+
+async function fetchDragons(dispatch, getState) {
+  const { dragons: currentDragons } = getState();
+
+  if (currentDragons.length === 0) {
+    const { data } = await axios.get('https://api.spacexdata.com/v3/dragons');
```

Comment:
We can probably make the api calls using a service as described under the section [Initialize Axios for React Hooks Redux CRUD HTTP Client](https://www.bezkoder.com/react-hooks-redux-crud/#Initialize_Axios).